Source/WebCore/platform/mac/ScrollbarsControllerMac.mm

```diff
@@ -954,11 +955,10 @@ - (void)invalidate
         verticalScrollbar->invalidate();
 
         NSScrollerImp *oldVerticalPainter = [m_scrollerImpPair verticalScrollerImp];
-        auto newVerticalPainter = retainPtr([NSScrollerImp scrollerImpWithStyle:newStyle controlSize:(NSControlSize)verticalScrollbar->widthStyle() horizontal:NO replacingScrollerImp:oldVerticalPainter]);
+        auto* verticalScrollbarMac = dynamicDowncast<ScrollbarMac>(verticalScrollbar);
+        verticalScrollbarMac->createScrollerImp(WTFMove(oldVerticalPainter));
```

Comment:
Do we know it's not null? What if the scrollbar is a custom scrollbar?

Reply:
There is a check for if its a custom scrollbar above this code.